static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -0,0 +1,619 @@
+          Choose a portal to refine search results to a particular area of study,
+          project, or organization.
+        </p>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'right',
+    disableScrolling: true,
+    styles: {
+      tooltip: {
+        width: '400px'
+      }
+    }
+  },
+  {
+    target: '.sidebar-section-body',
+    content: (
+      <div>
+        <StepCounter currentStep={stepIndex} />
+        <p className="tour-content">
+          Refine your search further using categories like Features, Keywords,
+          Platforms, Organizations, etc.
+        </p>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'right-start',
+    disableScrolling: true,
+    styles: {
+      tooltip: {
+        width: '400px'
+      }
+    }
+  },
+  {
+    target: '.panel-section',
+    content: (
+      <div>
+        <StepCounter currentStep={stepIndex} />
+        <p className="tour-content">
+          A high-level description is displayed for each search result to help you find
+          the right data, including a summary, temporal range, and information about
+          capabilities. To view more information about a collection, click the
+          {' '}
+          <EDSCIcon className="text-icon" icon={FaInfoCircle} />
+          {' '}
+          icon.
+        </p>
+        <p className="tour-content">
+          Add granules to a project and customize options before accessing the data.
+          To add a collection to your project, click the
+          {' '}
+          <EDSCIcon className="text-icon" icon={FaPlus} />
+          {' '}
+          icon.
+          To add individual granules to a project, click on a search result to view and
+          add its granules.
+        </p>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'right',
+    styles: {
+      tooltip: {
+        width: '400px'
+      }
+    },
+    floaterProps: {
+      disableFlip: true,
+      offset: 10
+    }
+  },
+  {
+    target: '.panels__handle',
+    content: (
+      <div>
+        <StepCounter currentStep={stepIndex} />
+        <p className="tour-content">
+          To make more room to view the map, the search results can be resized by clicking
+          or dragging the bar above. The panel can be hidden or shown by clicking the
+          handle or using the
+          {' '}
+          <kbd>]</kbd>
+          {' '}
+          key.
+        </p>
+        <div className="tour-info-box">
+          <p>
+            All keyboard shortcuts can be displayed by pressing the
+            {' '}
+            <kbd>?</kbd>
+            {' '}
+            key at any time.
+          </p>
+        </div>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'right',
+    styles: {
```

Comment:
we are gonna want to move alot of the repeated css into a classname also the bootstrap classes should be favored since we don't have to maintain them